src/wovnio/wovnphp/Headers.php

```diff
@@ -392,6 +402,15 @@ public function removeLang($uri, $lang = null)
 
         $lang_code = $this->store->convertToCustomLangCode($lang);
         $default_lang = $this->store->settings['default_lang'];
+        $url_pattern_name = $this->store->settings['url_pattern_name'];
+        $custom_domain_langs = $this->store->settings['custom_domain_langs'];
+        if ($url_pattern_name == 'custom_domain' && isset($custom_domain_langs)) {
+            $current_lang_domain = array_search($lang_code, $custom_domain_langs);
+            $default_lang_domain = array_search($default_lang, $custom_domain_langs);
+            $current_lang_path = URL::parseUrlStr($current_lang_domain)['path'];
+            $default_lang_path = URL::parseUrlStr($default_lang_domain)['path'];
+            return str_replace($current_lang_path, $default_lang_path, $uri);
```

Comment:
Look at https://github.com/WOVNio/Wovn.cs/blob/master/wovn.cs/Helpers/URL.cs#L109

This should be using the same function as addLangCode. This looks like it is duplicated. 

Do not be afraid to create individual functions for custom domain. I would like to see a similar `ChangeToNewCustomDomainLanguage` function.

Reply:
I'm trying to use removeLangCode in Url.php from Headers.php.
And I will replace url host and path separately.